README.md

```diff
@@ -95,12 +95,10 @@ Astro is free, open source software made possible by these wonderful sponsors.
 [❤️ Sponsor Astro! ❤️](https://github.com/withastro/.github/blob/main/FUNDING.md)
 
 <p align="center">
-  <a target="_blank" href="https://github.com/sponsors/withastro">
+  <a target="_blank" href="https://opencollective.com/astrodotbuild">
 
-[![Astro's sponsors.](https://astro.build/sponsors.png "Astro's sponsors.
-Platinum sponsors: Vercel, storyblok, Netlify, Ship Shape, Google Chrome
-Gold sponsors: ‹div›RIOTS, DEEPGRAM, Transloadit, CloudCannon
-Sponsors: Monogram, Qoddi, Dimension")](https://github.com/sponsors/withastro)
+[![Astro's sponsors.](https://astro.build/sponsors.png "Sponsor logos
```

Comment:
Sorry, maybe it's because I haven't had my coffee, but does this Markdown syntax *also* contain a link, which goes to sponsors vs open collective?

Reply:
You can check the rendered file and see if that works for us

https://github.com/withastro/astro/blob/chore%2Fsponsors-link/README.md

Reply:
Thanks, yeah the rendered file is still taking me to GitHub sponsors